homework/04LessonFour.js

```diff
@@ -87,7 +96,7 @@ var nestedObjects = {
 // What is an associative array? What is the difference between an array and an associative array?
 //
 // Your Answer Goes Here:
-//
+// Associative arrays are also called object. They are similar to arrays except they used named indexes instead of numbered indexes.
```

Comment:
Correct. **_Associative_** arrays have named indexes with values _**associated**_ to them.